static/css/components/searchbox.less

```
@@ -0,0 +1,41 @@
+/**
+ * Search button and text input
+ * [TODO: Link to documentation]
+ */
+
+.searchbox {
+  background-color: @white;
+  border: 1px solid @dark-beige;
+  border-radius: .3em;
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+  padding: .2%;
+  padding-left: .5%;
+  width: 50%;
+  height: 30px;
+}
+
+.searchbox__btn-icon {
+  background: url(/static/images/search-icon.svg) center no-repeat;
+  width: 35px;
+  height: 45px;
+  cursor: pointer;
+  border: 0;
+}
+
+.searchbox__input {
+  margin: 0 !important;
+  border: 0;
+  background: transparent;
+  color: @grey !important;
+  flex-grow: 1;
+  font-size: 1em !important;
+  height: 30px;
+  max-width: 94% !important;
+}
+@media (max-width: 500px) {
```

```suggestion
@media (max-width: @width-breakpoint-mobile) {
```
Lets use the variable here.